tests/pygments_ansi_color_test.py

```diff
@@ -105,6 +108,9 @@ def test_simple_colors():
         (Color.Bold, 'bold text\n'),
         (Color.Bold.BGYellow, 'bold from previous line with yellow bg\n'),
         (Color.Bold, 'bg color turned off\n'),
+        (Color.Bold.BrightGreen, 'fg bright green\n'),
+        (Color.Bold.BCBrightRed, 'bg bright red\n'),
```

Comment:
is this a typo? how is this passing

Reply:
My testing environment had an issue and didn't catch this. Fixed in 20d5fbe.